soql-server-pg/docker/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM socrata/java
+
+# TODO expose JMX
+EXPOSE 6090 6091
+
+ENV SERVER_ROOT /srv/soql-server-pg
+ENV SERVER_ARTIFACT soql-server-pg-assembly.jar
+ENV SERVER_CONFIG soql-server-pg.conf
+
+# defaults
+ENV ENABLE_GRAPHITE false
+ENV GRAPHITE_HOST 0.0.0.0
+ENV GRAPHITE_PORT 0
+ENV JAVA_XMX 512m
```

Comment:
The intent is it is a default that someone saying "oh, I just want to try running this" can reasonably use.  eg. developer running locally.  If 512 megs isn't enough for that then we should certainly bump it, but it feels like it should be reasonable.

The "this makes sense for a production environment" number is higher, but is a more specific scenario so I don't think it deserves to be the default.